src/skmultiflow/drift_detection/hddm_w.py

```diff
@@ -5,6 +5,8 @@
 
 class HDDM_W(BaseDriftDetector):
     """
+    Drift Detection Method based on Hoeffding’s bounds with moving weighted average-test.
```

Comment:
Should not "bounds" be singular in this case?

Reply:
Singular and plural work on this case (bound-calculation is the same, bound-value**s** change with the observed data). Here we use the plural to match the paper.

Reply:
I got your point. Agreed